### PR TITLE
Feat/catalog status filter

### LINKED ISSUE #1433: [2.2] Add new status filter and persist the filter in the URL
https://www.notion.so/decentraland/Unified-Markets-65195514608c429fbba4e0d0884fde3c?pvs=4#0607185f58984d879c759477ef7d9694
Create ui filter and apply the status to the URL parameters so it gets to the new endpoint of listings as the sortBy condition.

### PR DESCRIPTION
**Pull request by juanmahidalgo**
Closes #1433

image

### CODE REVIEW

File: webapp/src/modules/routing/selectors.ts [hunk 1]
```diff
@@ -180,6 +180,11 @@ export const getRarities = createSelector<RootState, string, Rarity[]>(
     )
 )
 
+export const getStatus = createSelector<RootState, string, string>(
```
Comment (cyaiox) [inline webapp/src/modules/routing/selectors.ts:183]: could you add a test for this selector?

File: webapp/src/components/AssetFilters/AssetFilters.container.ts [hunk 2]
```diff
@@ -67,6 +69,9 @@ const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
         ? values.maxEstateSize || ''
         : getMaxEstateSize(state),
     rarities: 'rarities' in values ? values.rarities || [] : getRarities(state),
+    status: ('status' in values
```
Comment (cyaiox) [inline webapp/src/components/AssetFilters/AssetFilters.container.ts:72]: the `status` value will have multiple selections?

```suggestion
    status: ('status' in values) ? values.status : getStatus(state) as AssetStatusFilter,
```

File: webapp/src/modules/routing/selectors.ts (hunk 1 above)
Comment (juanmahidalgo) [reply webapp/src/modules/routing/selectors.ts]: added!

File: webapp/src/modules/routing/selectors.ts [hunk 3]
```diff
@@ -21,12 +21,14 @@ import {
...
   getURLParamArray_nonStandard
 } from './search'
-import { BrowseOptions, SortBy } from './types'
+import { BrowseOptions, SortBy, SortByOption } from './types'
 import { locations } from './locations'
 import { AssetType } from '../asset/types'
 import { getAddress as getWalletAddress } from '../wallet/selectors'
 import { getAddress as getAccountAddress } from '../account/selectors'
 import { isLandSection } from '../ui/utils'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
```
Comment (cyaiox) [inline webapp/src/modules/routing/selectors.ts:30]: np: do you mind moving this import above?

File: webapp/src/components/AssetFilters/StatusFilter/StatusFilter.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+import { InfoTooltip } from '../../InfoTooltip'
+import './StatusFilter.css'
+
+export type StatusFilterFilterProps = {
+  status?: AssetStatusFilter
+  onChange: (value: BrowseOptions) => void
+  defaultCollapsed?: boolean
+}
+
+export const StatusFilter = ({
```
Comment (cyaiox) [inline webapp/src/components/AssetFilters/StatusFilter/StatusFilter.tsx:15]: should this component type be `React.FC`?

File: webapp/src/modules/routing/selectors.ts (hunk 3 above)
Comment (juanmahidalgo) [reply webapp/src/modules/routing/selectors.ts:30]: done!

**Review by cyaiox — APPROVED** [review]
🚀

**Pull request merged**